python/nano/test/onnx/pytorch/test_inc_onnx.py

```diff
@@ -240,6 +241,51 @@ def hello():
         with pytest.raises(AttributeError):
             onnx_model.width
 
+    def test_onnx_inc_default_values(self):
+        # default bool values
+        class Net(nn.Module):
+            def __init__(self):
+                super().__init__()
+            def forward(self, x, a=True, b=False):
+                if a:
+                    return x+1
+                if b:
+                    return x-1
+                return x
+
+        model = Net()
+
+        data = torch.rand(1,3,1,1)
+        result_true = model(data)
+        # sample with only required parameters (in a tuple)
+        accmodel = InferenceOptimizer.quantize(model,
+                                               accelerator="onnxruntime",
+                                               calib_data=torch.rand(2,3,1,1))
+        result_m = accmodel(data)
+
+        # sample with only required parameters (in a tuple)
+        accmodel = InferenceOptimizer.quantize(model,
+                                               accelerator="onnxruntime",
+                                               calib_data=torch.rand(2,3,1,1),
+                                               input_sample=(torch.rand(2,3,1,1), False, True))
+        result_m = accmodel(data)
+
+        # default bool values
+        class Net(nn.Module):
+            def __init__(self):
+                super().__init__()
+            def forward(self, x, a=3):
+                return x + a
+        model = Net()
+
+        data = torch.rand(1,3,1,1)
+
+        # sample with only required parameters (in a tuple)
+        accmodel = InferenceOptimizer.quantize(model,
+                                               accelerator="onnxruntime",
+                                               calib_data=(torch.rand(2,3,1,1), 5))
+        result_m = accmodel(data, np.array([5]))  # TODO: make this 5
+
```

Comment:
For the case I met [error](https://github.com/analytics-zoo/nano/issues/225), the use case is:
```python
elif method == 'onnxruntime_int8_qlinear':
    model = InferenceOptimizer.quantize(model=model,
                                        precision='int8',
                                        accelerator='onnxruntime',
                                        calib_data=input_sample,
                                        input_sample=input_sample,
                                        method='qlinear',
                                        thread_num=4, # need to specify to make sure onnxruntime is using the correct thread
                                        dynamic_axes=dynamic_axes)

elif method == 'onnxruntime_int8_integer':
    model = InferenceOptimizer.quantize(model=model,
                                        precision='int8',
                                        accelerator='onnxruntime',
                                        calib_data=input_sample,
                                        input_sample=input_sample,
                                        method='integer',
                                        thread_num=4, # need to specify to make sure onnxruntime is using the correct thread
                                        dynamic_axes=dynamic_axes)
``` 
in which the `input_sample` is a simple 4d tensor, and model's forward method is like:
```python
 def forward(self, im, augment=False, visualize=False, val=False):
```
I am not quiet understand the use case that do not include a `input_sample` in the `quantize` function, and why passing `(data, np.array([5]))` in the `accmodel`.

Reply:
So the `quantize` will first use `input_sample`(if valid, or `calib_data` will be used) to trace the model and then use `calib_data` to calib the model during the quantization. A special case should be there are `NoneType` or `bool` class parameter in the `forward`, if users want to change the default values of these parameter, they are required to put them in `input_sample`.

Onnxruntime will only take numpy ndarray as input for the prediction, that's why we need to use `np.array([5])` here. We should make `5` available as well.

It's quite tricky, we will put them in trouble shooting page for now.